tests/acceptance/stepDefinitions/previewContext.js

```diff
@@ -10,6 +19,27 @@ When('the user/public views the file {string} in the mediaviewer using the webUI
   return offCanvasPage.openMediaViewer()
 })
 
+When('the user navigates to the next media resource using the webUI', function() {
+  return offCanvasPage.nextMediaResource()
+})
+
+When('the user navigates to the previous media resource using the webUI', function() {
+  return offCanvasPage.previousMediaResource()
+})
+
+When('the user closes the media resource using the webUI', function() {
+  return offCanvasPage.closeMediaResource()
+})
+
+When('the user downloads the media resource using the webUI', function() {
+  return offCanvasPage.downloadMediaResource()
+})
+
 Then('the file {string} should be displayed in the mediaViewer', function(fileName) {
   return offCanvasPage.waitForMediaViewerLoaded(fileName)
 })
+
+Then('the file {string} should not be displayed in the mediaViewer', async function(fileName) {
```

Comment:
what is the difference between this step and the one above?

Reply:
> should and should not
one checks for the `displayed` media file and another for `not displayed`!